ci/build-package.ps1

```diff
@@ -0,0 +1,24 @@
+
+param(
+    [string]$ProjectDir = ".",
+    [string]$Name,
+    [string]$Version,
+    [Parameter(Mandatory=$true)]
+    [string]$JavaGpgKeyPassphrase,
+    [Parameter(Mandatory=$true)]
+    [string]$CodeSigningCert,
+    [Parameter(Mandatory=$true)]
+    [string]$JavaPGP,
+    [Parameter(Mandatory=$true)]
+    [string]$CodeSigningCertAlias,
+    [Parameter(Mandatory=$true)]
+    [string]$CodeSigningCertPassword,
+    [Parameter(Mandatory=$true)]
+    [string]$MavenSettings
+)
+
+
+./java/build-package.ps1 -RepoName "location-java" -ProjectDir $ProjectDir -Name $Name -Version $Version -JavaGpgKeyPassphrase $JavaGpgKeyPassphrase -CodeSigningCert $CodeSigningCert -JavaPGP $JavaPGP -CodeSigningCertAlias $CodeSigningCertAlias -CodeSigningCertPassword $CodeSigningCertPassword -MavenSettings $MavenSettings
```

Comment:
Instead of having "location-java" here, you can add `$RepoName` as a parameter, and it will be available to use.
Not a problem here, but will prevent any silly errors if this is copy/pasted.

Reply:
Same with a couple of other ones, but I'll leave as a single comment